patches/minecraft/net/minecraft/world/WorldEntitySpawner.java.patch

```diff
@@ -50,7 +81,18 @@
                                                      {
                                                          continue label134;
                                                      }
-@@ -219,6 +223,13 @@
+@@ -175,8 +185,10 @@
+                         }
+                     }
+                 }
++                else { eventBuilder.creatureTypeData.put(enumcreaturetype, new net.minecraftforge.event.world.FoundChunksForSpawningEvent.CreatureTypeData()); }
```

Comment:
Same here, this can be expanded, brace on new line, >code< , brace.

Reply:
I'd love to, but https://github.com/MinecraftForge/MinecraftForge/wiki/If-you-want-to-contribute-to-Forge says "However, to keep changes to .java.patch-files as minimal as possible. Try to write that statement to one single line:"

But if that's out of date or if I'm misinterpreting it, I'm happy to make the change.